forge-gui/res/cardsfolder/upcoming/candlekeep_inspiration.txt

```diff
@@ -0,0 +1,8 @@
+Name:Candlekeep Inspiration
+ManaCost:4 U
+Types:Sorcery
+A:SP$ AnimateAll | ValidCards$ Creature.YouCtrl | Power$ X | Toughness$ X | SpellDescription$ Until end of turn, creatures you control have base power and toughness X/X, where X is the number of cards you own in exile and in your graveyard that are instant cards, are sorcery cards, and/or have an Adventure.
+SVar:X:Count$ValidGraveyard Instant.YouOwn,Sorcery.YouOwn,Card.AdventureCard+YouOwn/Plus.Y
```

Comment:
I think `SVar:X:Count$ValidExile,Graveyard` should work already

Reply:
Just confirmed this, thanks for the tip. Fixed